docs/spec/consensus/light-client.md

```diff
@@ -1,113 +1,145 @@
+
+## Lite client initialization
+
+Lite client is initialized using a trusted socially verified validator set hash. This is part of the security model for
+proof of stake blockchains (see https://github.com/tendermint/tendermint/issues/1771). We assume that initialization 
+procedure receive as input the following parameters: 
+
+- height h: height of the blockchain for which trusted validator set hash is provided and
+- trusted validator set hash.
+
+Given this information, light client initialization logic will call `Validators` method on the node it is connected to 
+obtain `ResultValidators` that contains validators that has committed the block h. Then we check if MerkleRoot
+of the validator set is equal to the trusted validator set hash. If verification failed, initialization exits with error, otherwise it proceeds. 
+
+Next step is determining if the block at hight h is correctly signed by the obtained validator set. This is achieved by 
+calling `Commit` method for a given height that returns `SignedHeader` that is a header together with the set of 
+`Precommit` messages that commit the corresponding block. Light client logic then needs to verify if the set of signatures defined the valid commit for the given block. If this verification fails, initialization exits with error, otherwise it proceeds. 
+
+Final step (this should actually be part of the second step) of the initialization procedure is ensuring that we operate within UNBONDING_PERIOD of the trusted validator set. This can be expressed with the following formula:
 
-For the purpose of this light client specification, we assume that the Tendermint Full Node exposes the following functions over
-Tendermint RPC:
+`header.Time + UNBONDING_PERIOD <= Now() - BFT_TIME_DRIFT_BOUND`.
+
+Note that outside this time window lite client cannot trust validator set as validators could potentially unbonded its stake so security of the lite client does not hold as they are not slashable for its actions. Therefore, they can eclipse client and cheat about the system state without risk of being punished for such misbehavior. 
```

Comment:
```suggestion
Note that outside this time window lite client cannot trust validator set as validators could potentially have unbonded their stake so security of the lite client does not hold as they are not slashable for its actions. Therefore, they can eclipse client and cheat about the system state without risk of being punished for such misbehavior. 
```

Reply:
I don't understand what eclipsing and the unbonding period have to do with one another. If the light client is eclipsed for the full duration of the unbonding period it doesn't matter whether the validators are unbonding or not, since the light client can't submit evidence anyways. Outside the unbonding period, it doesn't matter if the client is eclipsed since the state machine will reject the evidence even if it is submitted.

Reply:
I assume here that being eclipsed is not permanent state of things as at any point in time you can decide to change full node you are connected to or decide to connect to new full node. In that sense there is a difference whether you are operating within unbonding period as you still have a chance of connected to correct node, detecting fork (for example) and submit evidence, compared to the case where you are outside unbonding period where there is no help if you are cheated.